apps/storybook-react/.storybook/main.ts

```diff
@@ -39,6 +39,10 @@ const config: StorybookConfig = {
           __dirname,
           '../../../packages/design-tokens',
         ),
+        '@metamask/design-system-react': path.resolve(
+          __dirname,
+          '../../../packages/design-system-react/src',
+        ),
```

Comment:
Fixes module resolution in Storybook by configuring Vite to use the source files directly from the `src` directory instead of the compiled `dist` files. This ensures all TypeScript types and exports are properly available during development, resolving the "missing TextProps export" error.

<img width="1512" alt="Screenshot 2024-11-21 at 5 55 02 PM" src="https://github.com/user-attachments/assets/b6eaee7d-8a16-46b8-896f-9bd5ea4964f1">